css-brochure-form/styles.css

```diff
@@ -0,0 +1,73 @@
+h1,
+label,
+button {
+  font-family: 'Brandon Grotesque', sans-serif;
+  text-transform: uppercase;
+  font-weight: 600;
+}
+
+h1 {
+  font-size: 15px;
+  color: rgb(236, 93, 87);
+  text-align: center;
+  margin: 14px auto 10px;
+  padding: 0;
+}
+
+form {
+  width: 383px;
+  margin: 0 auto;
+  padding: 0;
+}
+
+div {
+  height: 73px;
+}
+
+label {
+  font-size: 11px;
+  color: rgb(43, 43, 43);
+  padding: 0 10px;
+}
+
+input,
+select {
+  border: 1px solid rgb(166, 170, 169);
+  border-radius: 5px;
+  height: 37px;
+  width: 383px;
+  font-size: 13px;
```

Comment:
`height` here is correct because you want the `content` to be of a certain height.